core/src/main/scala/sttp/tapir/Endpoint.scala

```diff
@@ -314,6 +314,9 @@ trait EndpointInfoOps[-R] {
   private[tapir] def withInfo(info: EndpointInfo): ThisType[R]
 
   def name(n: String): ThisType[R] = withInfo(info.name(n))
+  def prefixName(n: String): ThisType[R] = withInfo(info.copy(name = Some(n + info.name.getOrElse(""))))
```

Comment:
this looks like an unrelated change - can we remove it from this PR?

Reply:
removed